docs/source/reference/yaml-spec.rst

```diff
@@ -35,7 +35,7 @@ describe all fields available.
       region: us-east-1
 
       # The zone to use (optional).
```

Comment:
Minor nit. Add: 
```
... Auto-failover will be disabled
      # if this is specified.
```
like L33-34?

L33's "The Auto-" -> "Auto-".

Reply:
Fixed!